tsconfig.json

```diff
@@ -26,11 +26,8 @@
       "#internal/nitro": [
         "./node_modules/nitropack/dist/runtime"
       ],
-      "#internal/nitro/app": [
-        "./node_modules/nitropack/dist/runtime/app"
-      ],
-      "#internal/nitro/utils": [
-        "./node_modules/nitropack/dist/runtime/utils"
+      "#internal/nitro/*": [
```

Comment:
Currently, (sadly) I cannot make any refactors to `/app` and `/utils` (which are explicit **internals**) because Nuxt is depending on them. This is dangerous and should be avoided. Please keep them explicit to these two and let's not extend to use anymore.